src/angle.rs

```diff
@@ -55,7 +55,7 @@ impl<T> Angle<T>
 where
     T: Rem<Output = T> + Sub<Output = T> + Add<Output = T> + Zero + FloatConst + PartialOrd + Copy,
 {
-    /// Returns this angle in the [0..2*PI[ range.
+    /// Returns this angle in the [0..2*PI) range.
```

Comment:
I prefer the more explicit `[from..to[` notation for exclusive ranges.

Reply:
I thought it was a typo. I've never seen that kind of notation before. Are there any other examples where it is used? It seems to me that notation with parentheses for exclusive boundaries and with square for inclusive is widely known and should be used

Reply:
Whether you're used to seeing `[a..b)` or  `[a..b[` depends largely on which country you had your math classes in. A lot of European countries for example use the latter. It's a pretty unambiguous notation so I'd like it to stay here.

Reply:
Ok